sympy/sets/sets.py

```diff
@@ -1968,6 +1968,14 @@ def __lt__(self, other):
             raise TypeError("Invalid comparison of set with %s" % func_name(other))
         return self.is_proper_subset(other)
 
+    def __eq__(self, other):
```

Comment:
For `set/frozenset`, it does not call `__eq__` of its elements. Like `tuple` does.

```
In [2]: tuple([Rational(3, 10), Rational(1, 5)]) == tuple([0.3, 0.2])
Out[2]: True

In [3]: frozenset([Rational(3, 10), Rational(1, 5)]) == frozenset([0.3, 0.2])
Out[3]: False
```

Any idea, what would be a good way to solve this?